cfgov/wagtailadmin_overrides/static/css/global.css

```diff
@@ -14,12 +15,27 @@ body:before {
 @media screen and (min-width: 800px) {
   body:before {
     box-sizing: border-box;
-    width: 200px;
-    padding: 0.5em;
     position: fixed;
+    overflow: hidden;
+    z-index: 1000;
     top: 0;
     left: 0;
-    z-index: 100;
+    width: 150px;
+    height: 35px;
+    margin-top: 0.5em;
+    padding: 0.5em;
+    line-height: 1.5;
+    transition-duration: .15s;
+    transition-property: inset-inline-start,padding-inline-start,width,transform,margin-top,min-height;
+    transition-timing-function: cubic-bezier(.4,0,.2,1);
+  }
+  button.sidebar__collapse-toggle {
+    z-index: 1001;
+  }
+  body:has(div.sidebar--slim):before {
```

Comment:
You have `sidebar--slim` here, which is a modifier in true [BEM](https://getbem.com). However, for whatever reason we don't use true BEM in our codebase, but our own version of BEM where modifiers are separated by `__` (which are confusingly the separator for elements in true BEM). Further up there is `sidebar__collapse-toggle`, so if this is true BEM that would be an element, which is what that looks like, however, if it is a modifier, we would have a mix of styles here.

What it looks like is happening here is Wagtail is using true BEM, while the rest of our codebase doesn't and probably should. I'm going to bring this up in dev-tango.